### DISCUSSION TITLE
Theme-Switcher

### DISCUSSION THREAD

**Discussion by shiv-2810** [ReactiveX/rxjs#7516] [Ideas / Feature request]
As a developer, I’ve noticed that working with bright screens for long hours can be really hard on the eyes. It can cause eye strain and discomfort, especially when working in low-light environments. Personally, I prefer dark mode because it feels easier on my eyes after working for a while.

I think it would be a great idea to add a theme switcher that allows users to switch between light mode and dark mode. This small feature can make a huge difference in how comfortable people feel using the app, especially for developers and users who spend long hours working with the screen.

**Comment by ujamX** [comment]
HI, @shiv-2810 

As you mentioned, a dark screen can be effective in reducing eye strain when working for long periods of time. 

However, RxJS is a library, a tool used in environments such as IDEs or code editors.

Therefore, providing themes from RxJS itself appears to be an out-of-scope feature.

We recommend that you change the theme settings of the IDE or editor you are using. Most IDEs and text editors support dark and light modes, and you can easily change them in preferences. 

This will allow you to work on all your code, including RxJS, in a more comfortable environment.
